meetings/2017-08-28/minutes.md

```diff
@@ -0,0 +1,69 @@
+# Benchmarking WG Meeting - Aug 28 
+## Links
+* Recording: https://www.youtube.com/watch?v=78Vvc7OqhME
+* Github issue: https://github.com/nodejs/benchmarking/issues/134
+
+## Agenda
+
+* Stand up
+* Actions from last meeting
+* Open issues review
+* Other business
+
+## Present
+* Michael Dawson (@mhdawson)
+* Kunal Pathak (@kunalspathak)
+* Gareth Ellis (@gareth-ellis)
+* Uttam Pawar (@uttampawar)
+* Benedikt Meurer
+
+## Agenda items
+### Stand up
+* Michael Dawson (@mhdawson)
+  * Session with Kunal/uttam to make progress on setting up DC-EIS.
+  * Followed up with Myles on startup fix, as Node version 8 startup has not yet recovered
```

Comment:
Maybe something @hashseed should be aware of?

Reply:
@MylesBorins had followed up and found that the fix was not backported propertly.  Last I looked it was still not back to the expected level in the benchmarking runs, but that was because a release had not yet been cut.